rustls/src/msgs/deframer.rs

```diff
@@ -734,6 +727,7 @@ const ALLOWED_CONSECUTIVE_EMPTY_FRAGMENTS_MAX: u8 = 32;
 #[cfg(feature = "std")]
 #[cfg(test)]
 mod tests {
+    /*
```

Comment:
Intentionally left commented out?

Reply:
Added a commit before this one that deletes the tests for the code that is then removed a few commits later.